sdk/eventhub/Azure.Messaging.EventHubs/samples/README.md

```diff
@@ -33,37 +33,40 @@ If you'd like to run samples that use [Azure.Identity](https://github.com/Azure/
 
 ## Available samples
 
-- [Hello world](./samples/Sample01_HelloWorld.cs)  
+- [Hello world](./Sample01_HelloWorld.cs)  
   An introduction to Event Hubs, illustrating how to create a client and explore an Event Hub.
 
-- [Create an Event Hub client with custom options](./samples/Sample02_ClientWithCustomOptions.cs)  
+- [Create an Event Hub client with custom options](./Sample02_ClientWithCustomOptions.cs)  
   An introduction to Event Hubs, exploring additional options for creating the different Event Hub clients.
 
-- [Publish an event batch to an Event Hub](./samples/Sample03_PublishAnEventBatch.cs)  
+- [Publish an event batch to an Event Hub](./Sample03_PublishAnEventBatch.cs)  
   An introduction to publishing events, using a batch with single event.  
+  
+- [Publish multiple event batches to an Event Hub](./Sample04_PublishMultipleEventBatches.cs)  
+  An example of publishing events using multiple batches.    
 
-- [Read events from an Event Hub](./samples/Sample04_ReadEvents.cs)  
+- [Read events from an Event Hub](./Sample05_ReadEvents.cs)  
   An introduction to reading all events available from an Event Hub.
 
-- [Publish an event batch using a partition key](./samples/Sample05_PublishAnEventBatchWithPartitionKey.cs)  
+- [Publish an event batch using a partition key](./Sample05_PublishAnEventBatchWithPartitionKey.cs)  
```

Comment:
This should be `./Sample06_PublishAnEventBatchWithPartitionKey.cs` (the sample was renumber from 5 to 6 in this PR)

Reply:
Thanks!   Missed that one.